host/esutils/client_os2.go

```diff
@@ -62,108 +63,118 @@ func (os2 *os2Client) PutIndexTemplate(t *testing.T, templateConfigFile, templat
 	template, err := os.Open(templateConfigFile)
 	require.NoError(t, err)
 
-	req := opensearchapi.IndicesPutIndexTemplateRequest{
-		Body: template,
-		Name: templateName,
+	req := osapi.IndexTemplateCreateReq{
+		Body:          template,
+		IndexTemplate: templateName,
 	}
 
 	ctx, cancel := createContext()
 	defer cancel()
-	resp, err := req.Do(ctx, os2.client)
+	resp, err := os2.client.IndexTemplate.Create(ctx, req)
 	require.NoError(t, err)
-	require.False(t, resp.IsError(), fmt.Sprintf("OS2 error: %s", resp.String()))
-	resp.Body.Close()
-
+	require.True(t, resp.Acknowledged, fmt.Sprintf("OS2 put index template unacknowledged: %s", resp.Inspect().Response.Body))
```

Comment:
```suggestion
	require.Truef(t, resp.Acknowledged, "OS2 put index template unacknowledged: %s", resp.Inspect().Response.Body)
```
minor, obviously, but eh why not.